codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSSwiftDependency.kt

```diff
@@ -4,45 +4,26 @@
  */
 package software.amazon.smithy.aws.swift.codegen
 import software.amazon.smithy.swift.codegen.SwiftDependency
+import software.amazon.smithy.swift.codegen.SwiftDependency.DistributionMethod
 
 class AWSSwiftDependency {
 
     companion object {
-        val AWS_SDK_IDENTITY = SwiftDependency(
-            "AWSSDKIdentity",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../aws-sdk-swift",
-            "AWSSDKIdentity",
-            SwiftDependency.DistributionMethod.SPR,
-        )
-        val AWS_SDK_HTTP_AUTH = SwiftDependency(
-            "AWSSDKHTTPAuth",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../aws-sdk-swift",
-            "AWSSDKHTTPAuth",
-            SwiftDependency.DistributionMethod.SPR,
-        )
-        val AWS_SDK_EVENT_STREAMS_AUTH = SwiftDependency(
-            "AWSSDKEventStreamsAuth",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../aws-sdk-swift",
-            "AWSSDKEventStreamsAuth",
-            SwiftDependency.DistributionMethod.SPR,
-        )
-        val AWS_CLIENT_RUNTIME = SwiftDependency(
-            "AWSClientRuntime",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../aws-sdk-swift",
-            "AWSClientRuntime",
-            SwiftDependency.DistributionMethod.SPR,
-        )
+        val AWS_SDK_IDENTITY = awsTargetNamed("AWSSDKIdentity")
+        val AWS_SDK_HTTP_AUTH = awsTargetNamed("AWSSDKHTTPAuth")
+        val AWS_SDK_EVENT_STREAMS_AUTH = awsTargetNamed("AWSSDKEventStreamsAuth")
+        val AWS_CLIENT_RUNTIME = awsTargetNamed("AWSClientRuntime")
+
+        private fun awsTargetNamed(name: String): SwiftDependency {
+            return SwiftDependency(
+                name,
+                "main",
+                "0.0.15",
+                "aws-sdk-swift",
+                "../../../aws-sdk-swift",
+                name,
+                DistributionMethod.SPR,
+            )
+        }
```

Comment:
Similar to in smithy-swift SwiftDependency, I have provided this helper to DRY up the many SwiftDependency declarations.

Again, ignore the version numbers for now.